src/controls/util.ts

```diff
@@ -25,7 +25,7 @@ export const NOT_ALLOWED_CURSOR = 'not-allowed';
  */
 export const getActionFromCorner = (
   alreadySelected: boolean,
-  corner: string,
+  corner: string | undefined,
```

Comment:
why not using the '?'

Reply:
its usually better to init all props even if undefined, rather than having some missing and some not
that way browser engine can do smarter optimizations when objects have similar shapes

Reply:
I don't remember
I agree with '?'

Reply:
Ahh because it can't be optional because it is not the last arg

Reply:
oh i thought it was an object.